include/fmt/format-inl.h

```diff
@@ -535,6 +541,17 @@ FMT_FUNC void vprint_rgb(rgb fd, string_view format, format_args args) {
   std::fputs(internal::data::RESET_COLOR, stdout);
 }
 
+FMT_FUNC void vprint_rgb(rgb fd, wstring_view format, wformat_args args) {
```

Comment:
Similarly, to prevent copy-paste move the implementation to a new function template, `internal::vprint_rgb`, and reuse here and in `string_view` overload.